apps/aecore/src/aecore_sup.erl

```diff
@@ -31,7 +31,8 @@ init([]) ->
     {ok, {{one_for_one, 5, 10}, [?CHILD(aec_peers, 5000, worker),
                                   %% XXX empty block instead ot genesis block,
                                   %% ticket #84
+                                 ?CHILD(aec_state, 5000, worker),
```

Comment:
@sennui I understand `aec_state` must be started after `aec_chain`, because `aec_state:init` calls `aec_chain`. Am I missing something? (See also [previous PR review](https://github.com/aeternity/epoch/pull/153#pullrequestreview-64347073).)

From OTP doc: ["When the supervisor is started, the child processes are started in order from left to right according to this list."](http://erlang.org/doc/man/supervisor.html#supervision_princ)

Reply:
you are correct, bug.